Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Rifles/gestio.yml

```
@@ -30,27 +30,27 @@
   - type: Wieldable
   - type: GunWieldBonus
     minAngle: -2
```

Minimum angle should be 1 degree.  If the Mosin has a minimum angle of 1 degree, this should too.
```suggestion
    minAngle: -1
```

Also, if this is a "marksman's rifle", why is the _unwielded_ minimum angle 2, making this most suitable for unwielded run-and-gun pot shots?  Maybe this was an oversight, but I do think it should use the (boring) defaults of 21 and -20.